### DISCUSSION TITLE
What should we name this project?

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#7] [Development]
Here are a couple of suggestions from our requirements meeting on 18 April 2022:

- **UUG** — UFS Unified Graphics
- **MUG** — Model Unified Graphics
- **VECTOR** — Visualization of Experimental CAMs for Transitions to Operations from Research

If you have any other ideas, please add them below.

**Comment by ian-noaa** [comment]
Two other non-acronym names that came up in discussions were:

* **Fido** — inspired by "RRFS/Rufus"
* **Arty** — inspired by "RT" in "3D-RTMA"

If desired, they could always be "backronym-ed" (e.g. - FIDO - "Friendly Interactive Data Observatory", etc...)

And one acronym:

* **MOVES** — MOdEl Visualization System (out of order, I know)

**Reply by christinaholtNOAA** [reply]
MOdel Visualization and Evaluation System. ;)

**Comment by cshartsough** [comment]
Display for Representing Atmospheric Graphics in Operations at NOAA. - DRAGON
